crates/protocol/hardforks/Cargo.toml

```diff
@@ -27,6 +27,8 @@ op-alloy-consensus.workspace = true
 
 [dev-dependencies]
 alloy-primitives = { workspace = true, features = ["rand", "arbitrary"] }
+revm.workspace = true
+op-revm.workspace = true
```

Comment:
Q: If this change includes this cargo module in the revm and op-revm workspace, what was the state before?

Reply:
This is just a dev dependency for the `hardforks` crate - added a test utility in this PR that actually deploys the bytecode and checks for the deployed codehash.